example/package.json

```
@@ -2,7 +2,10 @@
   "name": "oscal-viewer",
   "homepage": ".",
   "version": "0.1.0",
-  "private": true,
+  "repository": "git://github.com/EasyDynamics/oscal-react-library.git",
```

We probably don't want to use the `git://` protocol. So based on [`package.json` docs for `repository`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository) we probably also need to specify `directory`.
```suggestion
  "repository": {
    "type": "git",
    "url": "https://github.com/EasyDynamics/oscal-react-library.git",
    "directory": "example"
  },
```

The reason I'd recommend against using `git://` is that it's a little antiquated and is read-only (at least on GitHub). For more information about this protocol and others, see https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_git_protocol. `https` is a good compromise because for GitHub, no auth is required for reads (but is for pushes), it works in a web browser as well as the git CLI, and it supports encryption in transit. So it's cool that our